config.toml

```diff
@@ -174,3 +174,15 @@ enable = false
 	name = "Discord"
 	url = "https://discord.gg/hG3yc6uAyC"
 	icon = "fab fa-discord"
+
+[[params.versions]]
```

Comment:
Could be adjusted like this because v0.7 is not released yet:
```yaml
[[params.versions]]
	fullversion = "latest"
	version = "latest"
	docsbranch = "main"
	url = "https://openfunction.dev"

[[params.versions]]
	fullversion = "v0.6.0"
	version = "v0.6.0"
	docsbranch = "v0.6"
	url = "https://v0-6.openfunction.dev"
```

Reply:
I use this PR for this issue https://github.com/OpenFunction/openfunction.dev/issues/166

Reply:
Ah, that's what I'm thinking
Thanks!